Marlin/Configuration_adv.h

```diff
@@ -482,6 +482,23 @@
 // The timeout (in ms) to return to the status screen from sub-menus
 //#define LCD_TIMEOUT_TO_STATUS 15000
 
+/**
+ * LED Control Menu
+ * Enable this feature to add LED Control to the LCD menu
+ */
+//#define LED_CONTROL_MENU
+#if ENABLED(LED_CONTROL_MENU)
+  #define LED_COLOR_PRESETS                 // Enable the Preset Color menu option
+  #if ENABLED(LED_COLOR_PRESETS)
+    #define LED_USER_PRESET_RED       255   // User defined RED value
+    #define LED_USER_PRESET_GREEN     128   // User defined GREEN value
+    #define LED_USER_PRESET_BLUE        0   // User defined BLUE value
+    #define LED_USER_PRESET_WHITE     255   // User defined WHITE value
+    #define LED_USER_PRESET_INTENSITY 255   // User defined intensity
+    //#define LED_USER_PRESET_STARTUP       // Have the printer display the user preset color on startup
```

Comment:
@Tannoo What is the normal startup state for the LEDs? I assume they are simply turned off by default. Is this option intended to have the LEDs set to a certain color whenever Marlin re-starts (as it does on host connect)? I didn't see any code added to Marlin's `init()` function as a result of this setting.